odl/discr/lp_discr.py

```diff
@@ -1248,6 +1267,275 @@ def discr_sequence_space(shape, exponent=2.0, impl='numpy', **kwargs):
+    nsamples : `int` or `tuple` of `int`
+        Number of samples per axis. For dimension >= 2, a tuple is
+        required.
+    exponent : positive `float`, optional
+        The parameter :math:`p` in :math:`L^p`. If the exponent is not
+        equal to the default 2.0, the space has no inner product.
+    interp : `str` or `sequence` of `str`, optional
+        Interpolation type to be used for discretization.
+        A sequence is interpreted as interpolation scheme per axis.
+
+            'nearest' : use nearest-neighbor interpolation
+
+            'linear' : use linear interpolation
+
+    impl : {'numpy', 'cuda'}, optional
+        Implementation of the data storage arrays
+    nodes_on_bdry : `bool` or `sequence`, optional
+        If a sequence is provided, it determines per axis whether to
+        place the last grid point on the boundary (True) or shift it
+        by half a cell size into the interior (False). In each axis,
+        an entry may consist in a single `bool` or a 2-tuple of
+        `bool`. In the latter case, the first tuple entry decides for
+        the left, the second for the right boundary. The length of the
+        sequence must be ``array.ndim``.
+
+        A single boolean is interpreted as a global choice for all
+        boundaries.
+        Default: `False`
+
+    dtype : dtype, optional
+        Data type for the discretized space
+
+            Default for 'numpy': 'float64' / 'complex128'
+
+            Default for 'cuda': 'float32'
+
+    order : {'C', 'F'}, optional
+        Ordering of the axes in the data storage. 'C' means the
+        first axis varies slowest, the last axis fastest;
+        vice versa for 'F'.
+        Default: 'C'
+    weighting : {'const', 'none'}, optional
+        Weighting of the discretized space functions.
+
+            'const' : weight is a constant, the cell volume (default)
+
+            'none' : no weighting
+
+    Notes
+    -----
+    The parameters ``min_corner``, ``max_corner``, ``nsamples`` and
+    ``cell_sides`` can be combined in the following ways (applies in
+    each axis individually):
+
+    **0 arguments:**
+        Return a copy of ``discr``
+
+    **1 argument:**
+        [min,max]_corner -> keep sampling but translate domain so it
+        starts/ends at ``[min,max]_corner``
+
+        nsamples/cell_sides -> keep domain but change sampling.
+        See `uniform_partition` for restrictions.
+
+    **2 arguments:**
+        min_corner + max_corner -> translate and resample with the same
+        number of samples
+
+        [min,max]_corner + nsamples/cell_sides -> translate and resample
+
+        nsamples + cell_sides -> error due to ambiguity (keep
+        ``min_corner`` or ``max_corner``?)
+
+    **3+ arguments:**
+        The underlying partition is uniquely determined by the new
+        parameters. See `uniform_partition`.
+
+    See also
+    --------
+    uniform_partition : underlying domain partitioning scheme
+
+    Examples
```

Comment:
Ha, so now I know where the border goes (roughly), can start bisecting :-)

Now that I see it again, I probably don't need to repeat all the cases between the example blocks. I'll reduce to one "simple" block and one block that does a bit of everything in different axes.